File: 프로그래머스/unrated/120862. 최댓값 만들기 （2）/최댓값 만들기 （2）.js
function solution(numbers) {
    var arr = numbers.sort((x,y) => x - y)
    const len = arr.length
    // return arr[len - 1] * arr[len - 2] > arr[0] * arr[1] ? arr[len - 1] * arr[len - 2] : arr[0] * arr[1]
    return Math.max(arr[0] * arr[1], arr[len - 1] * arr[len - 2])
}

File: 프로그래머스/0/120862. 최댓값 만들기 （2）/최댓값 만들기 （2）.kt
import kotlin.math.*

class Solution {
    fun solution(numbers: IntArray): Int {
        val sortedArr = numbers.sorted()
        return max((sortedArr[0] * sortedArr[1]), (sortedArr[sortedArr.lastIndex] * sortedArr[sortedArr.lastIndex - 1]))
    }
}